src/index.css

```diff
@@ -77,3 +80,123 @@ form {
   display: flex;
   justify-content: space-between;
 }
+
+.add-btn {
+  background-color: #0290ff;
+  color: #fff;
+  font-size: 20px;
+  border: none;
+  height: 40px;
+  width: 150px;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.input-text {
+  width: 30%;
+  border-radius: 5px;
+  border: 1px solid gray;
+  padding-left: 15px;
+  font-size: 18px;
+  font-weight: 200;
+}
+
+.action-btn {
+  padding: 0 8px;
+  border: none;
+  background-color: #fff;
+  font-size: 18px;
+  color: #4386bf;
+}
+
+.remove-btn {
+  border-left: 2px solid #e8e8e8;
+  border-right: 2px solid #e8e8e8;
+}
+
+.prog-btn {
+  background-color: #0290ff;
+  color: #fff;
+  font-size: 16px;
+  border: none;
+  height: 40px;
+  width: 200px;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.book-update {
+  padding-left: 40px;
+  border-left: 2px solid #e8e8e8;
+}
+
+.current {
+  font-weight: 300;
+  color: #121212;
+  margin-bottom: 10px;
+}
+
+.chapter {
+  margin-bottom: 10px;
+  font-size: 1rem;
+  letter-spacing: -0.4px;
+}
+
+.title {
+  margin: 10px 0;
+  font-size: 1.375rem;
+  letter-spacing: -0.2px;
+}
```

Comment:
- [ ] Good work styling the app :clap: , as mentioned in the project requirements `Your final design should match Zeplin's design (this includes all elements + the same styling)` let's try to make our design as close to the original one as possible, I suggest some fixes below which we can implement in order to achieve this:
  - Let's use the correct font-weight for our book's title which is 700
  - you can use the same color from the design for the author'd name `color: #4386bf;`
  - we can also make the progress percentage i.e. `<p class="percent-complete">64%</p>` have the same size as in 
   the design which is 2rem

In addition to this you can refer to [the project created by one of Microverse students - Usman Suleiman Babakolo](https://pedantic-beaver-cb923b.netlify.app/#/). where you can inspect and get the exact styling, I encourage you to use the above design and try to match our design pixel perfect :+1: 

current design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/214917182-b5a14501-4132-4f27-a88b-c7967c830709.png)

Zeplin design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/214917298-e0a6ca68-817b-4c7a-bd90-16e491275f94.png)